EventSubscriber/LogRequestSubscriber.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ETSGlobal\LogBundle\EventSubscriber;
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\NullLogger;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+
+class LogRequestSubscriber implements LoggerAwareInterface
+{
+    use LoggerAwareTrait;
+    use LogTrait;
+
+    public function __construct()
+    {
+        $this->logger = new NullLogger();
+    }
+
+    public function __invoke(RequestEvent $event): void
+    {
+        memory_reset_peak_usage();
+
+        $request = $event->getRequest();
+        $request->attributes->set('__start_time', microtime(true));
+
+        $isContentExceed = $this->isContentExceedMaxSize($request->getContent());
+
+        if (!$isContentExceed) {
+            $this->logger->info(
+                'Start request',
+                [
+                    'method' => $request->getMethod(),
+                    'request_uri' => $request->getRequestUri(),
+                    'headers' => $request->headers->all(),
+                    'query' => $request->query->all(),
+                    'request' => $request->request->all(),
+                    'content' => $request->getContent(),
+                    'memory' => memory_get_usage(),
+                ],
+            );
+
+            return;
+        }
+
+        $this->logger->warning(
+            'Start request (long request content)',
+            [
+                'method' => $request->getMethod(),
+                'request_uri' => $request->getRequestUri(),
+                'headers' => $request->headers->all(),
+                'query' => $request->query->all(),
+                'request' => $request->request->all(),
+                'content' => $this->splitContent($request->getContent()),
+                'memory' => memory_get_usage(),
+            ],
+        );
+    }
```

Comment:
no need to duplicate everything just to change the message?

```suggestion
        $this->logger->warning(
           $isContentExceed ? 'Start request (long request content)' : 'Start request',
            [
                'method' => $request->getMethod(),
                'request_uri' => $request->getRequestUri(),
                'headers' => $request->headers->all(),
                'query' => $request->query->all(),
                'request' => $request->request->all(),
                'content' => $this->splitContent($request->getContent()),
                'memory' => memory_get_usage(),
            ],
        );
    }
```

Reply:
if string < max I want log level info and if string > max I want warning, I don't want to use dynamique call (perf issue)